app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryEstimate.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { isFeature } from '../../../models/beta/story';
+
+export class ExpandedStoryEstimate extends React.Component {
+  editStory(event) {
+    const newValue = event.target.value;
+    
+    this.props.onEdit({ estimate: newValue });
+  };
+
+  render() {
+    const { project, story } = this.props;
+
+    return (
+      <div className="Story__section">
+        <div className="Story__section-title">
+          { I18n.translate('activerecord.attributes.story.estimate') }
+        </div>
+
+        <select
+          defaultValue={story.estimate}
+          className="form-control input-sm"
+          onChange={(event) => this.editStory(event)}
```

Comment:
What do you think about turning this whole component in a function component and turn this line into:

```jsx
onChange={(event) => props.editStory({ estimate: event.target.value })}
```